.gitignore

```diff
@@ -285,6 +285,7 @@ distribute/*
 cmake_build
 .cmake_build
 cmake-build-debug
+cmake-build-debug-system
```

Comment:
Who is creating this directory? CC: @kgpai

Reply:
That's my bad. But I found why I have this new local folder.

I created a new CMake profile named "Debug-System" and apply another toolchain in order to customize my C++ compiler in my Clion. That's why the Clion created another `cmake-build-debug-system` folder for this new profile.
<img width="902" alt="image" src="https://user-images.githubusercontent.com/522549/211935077-5a218d82-ee70-4217-8e25-0505762ae08a.png">
I will revert this change. As a best practice, we should always use the **Default** CMake profile in Clion.